### PR TITLE
3rd party organizations on landing page

### LINKED ISSUE #78: 3rd party organizations on landing page
Some ISTQB products are developed in cooperation with 3rd parties, which are then mentioned on a landing page. 
Examples can be seen in https://istqb-main-web-prod.s3.amazonaws.com/media/documents/ISTQB_CT-AI_Syllabus_v1.0_mghocmT.pdf or https://istqb-main-web-prod.s3.amazonaws.com/media/documents/ISTQB-CT-UT_Syllabus_v1.0_2018.pdf (ignore copyright Notice on the landing page)

The current landing page layout is challenging with the space. But most of the syllabi has shorter name than ATLaS, and there is usually only one 3rd party to be mentioned. Also, we can tweak the layout a bit in case of a 3rd party.

Acceptance criteria:

- Landing page can be adjusted to mention 3rd party with text: "_Provided by <name of the 3rd party_"
- 3rd party logo can be added
- Multiple 3rd parties can be added
- The change could require TeX code update, so this doesn't need to be purely YAML or MD files change.
- There are no other changes trough the syllabus in case there is 3rd party involved.

### PR DESCRIPTION
**Pull request by Witiko**
This PR adds support for 3rd party organizations on the landing page and closes #78.

After this PR, authors can specify third-party organizations in file `metadata.yml` as follows:

``` yml
provided-by:
# 3rd-party organizations with logos
- name: Alliance for Qualification
  logo: a4q-logo
- name: Artificial Intelligence United
  logo: aiu-logo
# 3rd-party organizations without logos
- name: Chinese Software Testing Qualifications Board  # or just:
- Korean Software Testing Qualifications Board
```

For the document `example.tex`, this produces the following landing page:

ISTQB_Example_Document-1

### CODE REVIEW

**Review by danopolan — APPROVED** [review]
Looks great.

File: istqb.cls [hunk 1]
```diff
@@ -304,30 +308,97 @@
...
+            % Several third party organizations
+            \par \vfill
+            \seq_use:Nnnn
+              \g_istqb_third_parties_seq
+              { ~and~ }
+              { ,~ }
+              { ,~and~ }
+          }
+        \par \vspace { 0.3in }
+        \leaders \vrule width \textwidth \vskip 1pt
```
Comment (danopolan) [inline istqb.cls:365]: I would remove the lower horizontal line before 3rd party logos. It looks weird with that many horizontal lines on a page.

**Review by danopolan — CHANGES_REQUESTED** [review]
Please remove the horizontal before 3rd party logos.

**Comment by Witiko** [comment]
Of course. Here is how the document looks after removing the last horizontal line on the landing page:

ISTQB_Example_Document-1

**Pull request merged**